apisix/plugins/consumer-restriction.lua

```diff
@@ -86,7 +86,11 @@ local fetch_val_funcs = {
 }
 
 local function is_include(value, tab)
-    for k,v in ipairs(tab) do
+    if not value then
+        return false
+    end
```

Comment:
I don't think the logic here is correct, if the blacklist exists and we can't get the user info, the request should be forbidden

Reply:
1.For `consumer`, if have no corresponding `consumer_id`, we can assume that the user is unauthorized, it is reasonable for us to reject this request.
2.For `service` and `consumer_group`, they are not necessary, it will make user confused if we reject this request directly when the `service_id` or `consumer_group_id` not exists.

Reply:
I think reject is acceptable, we couldn't let the request go if it can't be verified.
We only need to optimize the error log in the issue.

Reply:
You are right, but it's too complicated to check every scenario, let's make the logic simple, the user who use the plugin should make the assigned type is present in every request.
So i think it's ok to optimize the related error log
```
Missing authentication or identity verification
```

Reply:
Ok, I will update it.